packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -6907,84 +6893,69 @@ export default class Meeting extends StatelessWebexPlugin {
   }
 
   /**
-   * Publishes a track.
+   * Publishes a stream.
    *
-   * @param {LocalTrack} track to publish
+   * @param {LocalStream} stream to publish
+   * @param {MediaType} mediaType of the stream
    * @returns {Promise}
    */
-  private async publishTrack(track?: LocalTrack) {
-    if (!track) {
-      return;
-    }
-
-    if (this.mediaProperties.webrtcMediaConnection) {
-      if (this.isMultistream) {
-        await this.mediaProperties.webrtcMediaConnection.publishTrack(track);
-      } else {
-        track.setPublished(true); // for multistream, this call is done by WCME
-      }
+  private async publishStream(stream: LocalStream, mediaType: MediaType) {
+    if (this.isMultistream && this.mediaProperties.webrtcMediaConnection) {
+      await this.sendSlotManager.publishStream(mediaType, stream);
     }
   }
 
   /**
-   * Un-publishes a track.
+   * Un-publishes a stream.
    *
-   * @param {LocalTrack} track to unpublish
+   * @param {MediaType} mediaType of the stream
    * @returns {Promise}
    */
-  private async unpublishTrack(track?: LocalTrack) {
-    if (!track) {
-      return;
-    }
-
+  private async unpublishStream(mediaType: MediaType) {
     if (this.isMultistream && this.mediaProperties.webrtcMediaConnection) {
-      await this.mediaProperties.webrtcMediaConnection.unpublishTrack(track);
-    } else {
-      track.setPublished(false); // for multistream, this call is done by WCME
+      await this.sendSlotManager.unpublishStream(mediaType);
     }
   }
 
   /**
-   * Publishes specified local tracks in the meeting
+   * Publishes specified local streams in the meeting
    *
-   * @param {Object} tracks
+   * @param {Object} streams
    * @returns {Promise}
    */
-  async publishTracks(tracks: LocalTracks): Promise<void> {
+  async publishStreams(streams: LocalStreams): Promise<void> {
     this.checkMediaConnection();
 
-    this.annotationInfo = tracks.annotationInfo;
+    this.annotationInfo = streams.annotationInfo;
 
     if (
-      !tracks.microphone &&
-      !tracks.camera &&
-      !tracks.screenShare?.audio &&
-      !tracks.screenShare?.video
+      !streams.microphone &&
+      !streams.camera &&
+      !streams.screenShare?.audio &&
+      !streams.screenShare?.video
     ) {
       // nothing to do
       return;
     }
 
     let floorRequestNeeded = false;
 
-    if (tracks.screenShare?.video) {
-      await this.setLocalShareVideoTrack(tracks.screenShare?.video);
-
-      floorRequestNeeded = this.screenShareFloorState === ScreenShareFloorStatus.RELEASED;
+    if (streams.screenShare?.audio) {
```

Comment:
this doesn't look right, see my other comments about share audio on transcoded meetings

Reply:
Updated this and also added a comment above like this,

> //Screenshare Audio is supported only in multi stream. So we check for screenshare audio presence only if it's a multi stream meeting

Done in #3053